hermeto/core/models/input.py

```diff
@@ -43,6 +44,22 @@ def show_error(error: "ErrorDict") -> str:
         location = " -> ".join(map(str, error["loc"]))
         message = error["msg"]
 
+        if "expected tags:" in message:
+            tags = message.split("expected tags:")[1].strip()
+            tags = [tag.strip("'") for tag in tags.split(", ")]
+            # For each tag, if there's an experimental version, use that instead
+            filtered_tags = []
+            for tag in tags:
+                if tag.startswith("x-"):
+                    filtered_tags.append(tag)
+                elif f"x-{tag}" not in tags:
+                    filtered_tags.append(tag)
```

Comment:
What happens when `x-{tag}` is in `tags`? The way the code is written it might look like a missing `else` case is an accident. You would want to avoid such ambiguity.